pandas/core/index.py

```diff
@@ -1398,15 +1396,34 @@ def sym_diff(self, other, result_name=None):
         the_diff = sorted(set((self.difference(other)).union(other.difference(self))))
         return Index(the_diff, name=result_name)
 
-    def get_loc(self, key):
+    def get_loc(self, key, method=None):
         """
         Get integer location for requested label
 
+        Parameters
+        ----------
+        key : label
+        method : {None, 'pad'/'ffill', 'backfill'/'bfill', 'nearest'}
+            default: exact matches only.
+            pad / ffill: find the PREVIOUS index value if no exact match.
+            backfill / bfill: use NEXT index value if no exact match
+            nearest: use the NEAREST index value if no exact match. Tied
+            distances are broken by preferring the larger index value.
```

Comment:
I suppose that one would maybe want to specify how things are tied (e.g. a side argument), but for a future PR